lib/android.dart

```diff
@@ -268,19 +238,15 @@ void saveNewImages(AndroidIconTemplate template, Image image,
 /// with the new icon name (only if it has changed)
 ///
 /// Note: default iconName = "ic_launcher"
-Future<void> overwriteAndroidManifestWithNewLauncherIcon(
-    String iconName, File androidManifestFile) async {
+Future<void> overwriteAndroidManifestWithNewLauncherIcon(String iconName, File androidManifestFile) async {
   // we do not use `file.readAsLinesSync()` here because that always gets rid of the last empty newline
-  final List<String> oldManifestLines =
-      (await androidManifestFile.readAsString()).split('\n');
-  final List<String> transformedLines =
-      transformAndroidManifestWithNewLauncherIcon(oldManifestLines, iconName);
+  final List<String> oldManifestLines = (await androidManifestFile.readAsString()).split('\n');
+  final List<String> transformedLines = transformAndroidManifestWithNewLauncherIcon(oldManifestLines, iconName);
   await androidManifestFile.writeAsString(transformedLines.join('\n'));
 }
 
 /// Updates only the line containing android:icon with the specified iconName
-List<String> transformAndroidManifestWithNewLauncherIcon(
-    List<String> oldManifestLines, String iconName) {
+List<String> transformAndroidManifestWithNewLauncherIcon(List<String> oldManifestLines, String iconName) {
```

Comment:
Since we aren't using this outside of this file, we can make this method private
```suggestion
List<String> _transformAndroidManifestWithNewLauncherIcon(List<String> oldManifestLines, String iconName) {
```

Reply:
fixed in 7e3b5a4